.pre-commit-config.yaml

```diff
@@ -21,6 +21,7 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
+        args: [--exclude=/semgrep/semgrep/output_from_core.py]
```

Comment:
I don't think this works. I've regenerated a few times this file and it got reformatted by some tools (black I guess).